### PR TITLE
Update Conda FAQ

### PR DESCRIPTION
**Pull request by afgane**
Readin this for the first time, I felt these changes make the text more clear.

### CODE REVIEW

File: doc/source/admin/conda_faq.rst [hunk 1]
```diff
@@ -18,25 +18,24 @@ Not only do Conda packages make tool dependencies more reliable and
...
-As a community, we have decided that Conda is the one that best fulfills
-our needs. The following are some of the crucial Conda features that led
+Conda is a package manager like ``apt-get``, ``yum``, ``pip``, ``brew`` or
+``guix``. We don't want to argue about the relative merits of various package
+managers here, in fact Galaxy supports multiple package managers and we welcome
+community contributions (such as implementing a Guix package manager or
+enhancing the existing brew support to bring it on par with Conda).
+
+As a community, we have decided that Conda is the one that best fulfills the
+project's needs. The following are some of the crucial Conda features that led
```
Comment (jmchilton) [inline doc/source/admin/conda_faq.rst]: I don't like the change on this line - I think the "project" is the code and it doesn't have needs and it should to some extent be agnostic. The community is the people - they have needs and conda is what works best for this community.

File: doc/source/admin/conda_faq.rst [hunk 2]
```diff
@@ -148,39 +145,73 @@ be configured in the ``dependency_resolvers_conf.xml`` and the ``galaxy.ini`` fi
...
 
-The Galaxy log will tell you which dependency resolution system is used
+This depends on your ``galaxy.ini`` setting. Galaxy will install the Conda
+package manager for you if you have enabled ``conda_auto_init``. If
+``conda_auto_install`` is enabled, Galaxy will install Trinity via Conda only
+when a Trinity job is launched and Trinity is not yet installed. With release
+16.07 you can see which dependencies are being used in the “Manage installed
+tools” section of the Admin panel and you can select whether or not to install
+Conda packages or Tool Shed package recipes when you install new tools there,
+even if ``conda_auto_install`` is disabled.
```
Comment (jmchilton) [inline doc/source/admin/conda_faq.rst]: This is a good a question and a fine answer - but I wonder if the answer should note that you can install dependencies along with tools right when installing a tool.

"In 16.07, out of the box when installing new tools you will be given the option to attempt installs of conda packages as well. ..."  Sort of moving the last part of this paragraph first. Not a big thing though.

File: doc/source/admin/conda_faq.rst [hunk 3]
```diff
@@ -234,11 +265,11 @@ and assist with all development. You will also see a few of us at this
...
 
 Don't be scared! Conda recipes are really simple to write. Conda also
-offers you, so called \`skeleton\` generators, generating recipes from
-pypi, cran, cpan for you (mostly) automatically.
+offers so called \`skeleton\` generators that generate recipes from
+pypi, cran, or cpan for you (mostly) automatically.
 
 
-9. Is there a way to convert traditional Tool Shed package recipes that are not yet in a Conda channel?
+11. Is there a way to convert traditional Tool Shed package recipes that are not yet in a Conda channel?
```
Comment (jmchilton) [inline doc/source/admin/conda_faq.rst:275]: This is out of order now right? 10 seems to come after this - or is that an artifact of the github diff.

File: doc/source/admin/conda_faq.rst (hunk 1 above)
Comment (afgane) [reply doc/source/admin/conda_faq.rst]: I'll update it; earlier formulation read a bit selfishly despite the initial community involvement...

Comment (martenson) [reply doc/source/admin/conda_faq.rst]: Please mind that this is adopted 'IUC letter to development community'. The community has all the reasons to be selfish here.

**Comment by lparsons** [comment]
Just a note, should also update the conda environment naming (https://github.com/galaxyproject/galaxy/pull/2891/files#diff-da3f510e46f823550300efe53a99fa62R136) to match https://github.com/galaxyproject/galaxy/commit/e9d81500caa3526200b204270aca175839f36f56

**Comment by martenson** [comment]
thanks @afgane !

**Comment by bgruening** [comment]
Thanks @afgane!

**Pull request merged**